ee/codegen/package.json

```diff
@@ -24,6 +24,7 @@
     "test": "vitest --run",
     "prepublishOnly": "npm run build && npm run gar-login",
     "test:update": "vitest --run -u",
+    "test:debug": "vitest --run --inspect-brk --no-file-parallelism",
```

Comment:
Helps to debug codegen tests. Will flesh out a README section when approach is stable